src/testhost/app.manifest

```diff
@@ -7,6 +7,9 @@
     <application>
       <!-- Windows 10 -->
       <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}"/>
+      <maxversiontested Id='10.0.18362.0'/>
```

Comment:
Two questions:
1) Have you confirmed that this doesn't prevent tests from running on RS5 (10.0.17763)?
2) Should newer versions be added as well? Like 22621 (22H2) and 22631 (23H2)?

Reply:
@MSLukeWest I've not tested this on older versions, but I understand this will have no effect on them. and newer versions should not be needed... @asklar can you help me explain why? I recall you (or Gov?) recommended multiple to support running on intermediate versions. I read [this bug](https://task.ms/33672605) but don't understand its implications or if it relates to this quirk or others. In [your sample](https://github.com/asklar/xaml-islands/blob/31c4e0c2e30eca9b90790dd83ed9062cd66bcba5/inc/xaml-islands.manifest#L10) you only specify one value `10.0.18226.0` (the value from the quirk that uses `ENABLED_VERSION_LT`). In my testing that works on testhost.exe.

```xml
          <QUIRK NAME="BlockXamlIslands" CODE_ID="262" ENABLED_VERSION_LT="10.0.18226.0" CONTACT_ALIAS="turekhe" ID="{57064B35-BE7D-4111-A3D0-FF077347F207}">
            <DESCRIPTION>
              Xaml Islands were first shipped in RS5. It was just in preview mode and an incomplete feature.
              However, Islands in 19H1 are more stable and complete. 
              Hence Islands are officially available in 19H1 and onward releases and are blocked on all prior releases.
            </DESCRIPTION>
          </QUIRK>
```

Reply:
the original bug I had filed is https://task.ms/33687330
IIUC you have to specify 18362 for xaml islands, and there's another value for segoe ui variable support. Adding a later value doesn't enable the features unlocked by an earlier value which is what makes this maxversiontested flag so confusing. @DHowett might remember this more

Reply:
specifying `18226` works in my testing.

Reply:
@MSLukeWest per the email thread I've updated this change to use `10.0.18362.0`, the first public release of the os after `10.0.18226.0`. 

its ready to go so please merge it.

Reply:
Approved, @nohwnd please merge for us